langwatch/src/pages/[project]/analytics/custom/index.tsx

```diff
@@ -1244,17 +1246,16 @@ function GraphTypeField({
         >
           <Select.Trigger>
             <Select.ValueText>
-              {(items) => {
```

Comment:
This is weird that we need to do this and accept no args. Can you just pass the children directly?